src/fs/dvfs/dvfs_util.c

```diff
@@ -135,8 +140,26 @@ int dvfs_destroy_inode(struct inode *inode) {
  */
 struct dentry *dvfs_alloc_dentry(void) {
 	struct dentry *d = pool_alloc(&dentry_pool);
-	if (d)
-		memset(d, 0, sizeof(struct dentry));
+	struct dentry *t = NULL;;
```

Comment:
This function is pretty short, I thought it would be okay.
"d" stands for "dentry", "t" stands for "temp".
But if you think variables should not be that short, I'll fix it.